src/System.Security.Permissions/src/System/Security/HostProtectionException.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace System.Security
+{
+    public partial class HostProtectionException : System.SystemException
+    {
+        public HostProtectionException() { }
+        // protected HostProtectionException(System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context) { }
+        public HostProtectionException(string message) { }
+        public HostProtectionException(string message, System.Exception e) { }
+        public HostProtectionException(string message, System.Security.Permissions.HostProtectionResource protectedResources, System.Security.Permissions.HostProtectionResource demandedResources) { }
+        public System.Security.Permissions.HostProtectionResource DemandedResources { get { return default(System.Security.Permissions.HostProtectionResource); } }
+        public System.Security.Permissions.HostProtectionResource ProtectedResources { get { return default(System.Security.Permissions.HostProtectionResource); } }
```

Comment:
Others may disagree with me, but I'd suggest we at least make the data passed to the constructor be propagated out of these properties.  That would mean adding fields to store these values in the ctor, and then returning the value of those fields here.  Or you could take advantage of the C# 6 feature that lets you declare get-only properties that can be assigned from the ctor, e.g.
```C#
public HostProtectionException(string message, System.Security.Permissions.HostProtectionResource protectedResources, System.Security.Permissions.HostProtectionResource demandedResources)
{
    DemandedResources = demandedResources;
    ProtectedResources = protectedResources;
}

public HostProtectionResource DemandedResources { get; }
public HostProtectionResource ProtectedResources { get; }
```